autonomy/data/Dockerfiles/agent/Pipfile

```diff
@@ -1,16 +0,0 @@
-[[source]]
```

Comment:
Removed because all in base now right? Where is the base image though? Shouldn't this just be moved there?

Reply:
https://github.com/valory-xyz/open-autonomy/pull/1221#discussion_r953603261